### PR TITLE
Adds links to job results in DukeDS, UI tweaks

### LINKED ISSUE #7: Include output directory link with jobs
- should show in job list for quick access
- Also show in job details

### LINKED ISSUE #8: Add some missing titles/headers for usability
- title/header when filling out a questionnaire
- add "jobs" link to nav bar (since it's not present)
- Add "logged in as" to navbar

### PR DESCRIPTION
**Pull request by dleehr**
- Adds links to results in DukeDS (project only) for finished jobs
- Updates wording and adds some descriptive/help text
- Changes job control buttons to not allow finished jobs to be restarted
- Minor markup/layout changes in navbar, jobs listing

Fixes #7 
Fixes #8

### CODE REVIEW

**Comment by dleehr** [comment]
Also adds a model `job-output-dir` and relationship from the **job**'s `output_dir` field.

Originally I assumed the nested JSON would just be stored as text, but Ember represents this as `[Object object]`. Since `output_dir` is really a related object, it was better to add a model.

Since this is the first instance of having a related model that's actually serialized/embedded in another object, it required a special case serializer

File: tests/unit/models/job-output-dir-test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { testRelationship } from '../../helpers/test-relationships';
+
+moduleForModel('job-output-dir', 'Unit | Model | job output dir', {
+  // Specify the other units that are required for this test.
+  needs: ['model:dds-project', 'model:job', 'model:dds-user-credential']
+});
+
+test('it exists', function(assert) {
+  let model = this.subject();
+  // let store = this.store();
```
Comment (johnbradley) [inline tests/unit/models/job-output-dir-test.js]: Is this commented out line just part of the ember generated cruft?

Comment (dleehr) [inline tests/unit/models/job-output-dir-test.js]: Yeah

**Comment by johnbradley** [comment]
Tests ran fine locally. I was able to click the DukeDS project link for finished jobs. Start, Restart, Cancel buttons enable/disable appropriately. I was able to see the "Your job is ready...Click Start to submit...." comment when looking at a New job. LGTM.

**Pull request merged**